src/PageContent/AssignVolunteers/Edit.js

```diff
@@ -0,0 +1,133 @@
+                                <label className="label-component details">Phone</label><br />
+                                <input name="phone" type="tel" pattern="[0-9]{3}-[0-9]{3}-[0-9]{4}" placeholder="xxx-xxx-xxxx" className="form-input" onChange={this.handlePhoneTwo.bind(this)} value={this.state.s2phone} required/><br />
+                                <label className="label-component details">Email</label><br />
+                                <input type="email" className="form-input" onChange={this.handleEmailTwo.bind(this)} value={this.state.s2email} required/>
+                            </div>
+
+                            <div className="form-child second-row">
+                                <h5 className="label-component" id="info">Email notifications will be sent to all deliverers.</h5>
+                            </div>
+                            <div className="form-child second-row form-buttons-container">
+                                <button type="button" className="form-button" onClick={this.props.handleCancelClick}>Cancel</button>
+                                <button type="submit" className="form-button confirm-button" onSubmit={this.handleConfirmClick.bind(this)}>Confirm</button>
+                            </div>
+
+
+                        </div>
+
+                    </form>
+                </div>
+            </div>
+        );
+
+    }
+
+    checkStudents(props) {
+        if(props.delivery.delivererGroup.deliverers[0]) {
+            this.setState({
+                s1name: props.delivery.delivererGroup.deliverers[0].name,
+                s1email: props.delivery.delivererGroup.deliverers[0].email,
+                s1phone: props.delivery.delivererGroup.deliverers[0].phone,
+            });
+            if(props.delivery.delivererGroup.deliverers[1]) {
+                this.setState({
+                    s2name: props.delivery.delivererGroup.deliverers[1].name,
+                    s2email: props.delivery.delivererGroup.deliverers[1].email,
+                    s2phone: props.delivery.delivererGroup.deliverers[1].phone,
+                });
+            }
+        }
+    }
+
+    handleNameOne(e) {
+        this.setState({s1name: e.target.value});
+    }
+
+    handlePhoneOne(e) {
+        this.setState({s1phone: e.target.value});
+    }
+
+    handleEmailOne(e) {
+        this.setState({s1email: e.target.value});
+    }
+
+    handleNameTwo(e) {
+        this.setState({s2name: e.target.value});
+    }
+
+    handlePhoneTwo(e) {
+        this.setState({s2phone: e.target.value});
+    }
+
+    handleEmailTwo(e) {
+        this.setState({s2email: e.target.value});
+    }
+
+    handleConfirmClick(e) {
+        e.preventDefault();
+        this.props.handleConfirmClick(this.state.s1name, this.state.s1phone, this.state.s1email, this.state.s2name, this.state.s2phone, this.state.s2email);
```

Comment:
here, if you change it to `<form>`, you should create the two object and pass them in:
```
var deliverer1 = {
    name: e.target.d1name.value,
    phone: e.target.d1phone.value,
    email: e.target.d1email.value,
}
var deliverer2 = {
    name: e.target.d2name.value,
    phone: e.target.d2phone.value,
    email: e.target.d2email.value,
}
this.props.handleConfirmClick(deliverer1, deliverer2)
```